### ISSUE TITLE
Flask 3.0.0

### ISSUE THREAD

**Issue by archerne** [h-janes/flask-sitemapper#10]
Can this be updated to work with Flask 3.0.0? Currently:
```
The conflict is caused by:
    The user requested flask==3.0.0
    flask-sitemapper 1.6.2 depends on flask<3.0.0 and >=2.2.2
```

**Comment by laike9m** [comment]
+1

**Comment by laike9m** [comment]
I installed Flask 3.0.0 and all tests can pass. I'll make a PR to upgrade project requirement, hope it can get accepted.

**Comment by h-janes** [comment]
thank you! really sorry for the wait, should be published as v 1.7.0 now

**Comment by laike9m** [comment]
Thank you!
